### ISSUE TITLE
Debug Failure. False expression: Token end is child end — For 'add missing super call' quick fix

### ISSUE THREAD

**Issue by mjbvz** [microsoft/TypeScript#32833]
**TypeScript Version:**  3.6.0-dev.20190810

**Search Terms:**
- quick fix 
- add missing super call

**Code**

For the TypeScript:

```ts
console.log(1)
console.log(2)
console.log(3)
console.log(4)
console.log(5)
console.log(6)

class A {}

class B extends A {
    constructor() {1;}
}
```

1. Trigger the `add missing super call` in the constructor of `B`

**Expected behavior:**
A quick fix is returned that changes the code to:

```ts
console.log(1)
console.log(2)
console.log(3)
console.log(4)
console.log(5)
console.log(6)

class A {}

class B extends A {
    constructor() {
        super();
        1;
    }
}
```

**Actual behavior:**
No quick fixes returned. See this TS server error in the console:

```
TypeScript Server Error (3.6.0-dev.20190810)
Debug Failure. False expression: Token end is child end
Error: Debug Failure. False expression: Token end is child end
    at processChildNode (tsserver.js:113342:34)
    at tsserver.js:113289:21
    at visitNode (tsserver.js:16956:24)
    at Object.forEachChild (tsserver.js:17202:24)
    at processNode (tsserver.js:113288:20)
    at processChildNode (tsserver.js:113348:21)
    at processChildNodes (tsserver.js:113399:48)
    at tsserver.js:113291:21
    at visitNodes (tsserver.js:16961:24)
    at Object.forEachChild (tsserver.js:17191:24)
    at processNode (tsserver.js:113288:20)
    at formatSpanWorker (tsserver.js:113103:17)
    at tsserver.js:113067:140
    at Object.getFormattingScanner (tsserver.js:111860:23)
    at Object.formatNodeGivenIndentation (tsserver.js:113067:31)
    at getFormattedTextOfNode (tsserver.js:115103:45)
    at format (tsserver.js:115081:52)
    at computeNewText (tsserver.js:115084:23)
    at tsserver.js:115054:89
    at Array.map (<anonymous>)
    at tsserver.js:115053:50
    at Array.map (<anonymous>)
    at Object.getTextChangesFromChanges (tsserver.js:115039:86)
    at ChangeTracker.getChanges (tsserver.js:115009:45)
    at Function.ChangeTracker.with (tsserver.js:114544:32)
    at Object.getCodeActions (tsserver.js:119357:64)
    at tsserver.js:115564:121
    at Object.flatMap (tsserver.js:583:25)
    at Object.getFixes (tsserver.js:115564:23)
    at tsserver.js:125578:35
```

**Comment by andrewbranch** [comment]
Related: #32862

**Comment by orta** [comment]
Looks like this has been fixed at some point in 3.7

![924F2704-FA1D-4731-A067-8F7EE1145078-42466-0003D0E4AB40B52F](https://user-images.githubusercontent.com/49038/69162189-4077a880-0aba-11ea-927f-b7a159cb756f.gif)
